packages/next/client/polyfill.js

```diff
@@ -0,0 +1,2 @@
+import 'whatwg-fetch'
+Object.assign = require('object.assign/polyfill')()
```

Comment:
I'd prefer to use `object-assign`, as it's a more lightweight dependency and what the React core team and CRA use. Is there any reason `object.assign` was chosen?

Reply:
@janicklas-ralph I think you mean `prop-types-exact`, `prop-types` uses `object-assign`.

See this frustrating issue: https://github.com/airbnb/prop-types-exact/issues/6. ljharb is hellbent on using his own package, despite `object-assign` being a better choice for the community. Once https://github.com/facebook/prop-types/issues/263 is fixed, `prop-types-exact` could be dropped anyway.

Reply:
Ah yes! Thanks for pointing it out. Fortunately Timer can up with an alternate approach that solves the original problem. So we'll be using `object-assign` throughout